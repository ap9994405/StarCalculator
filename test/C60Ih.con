60
    1      2    4    8
    2      1    3   11
    3      2    5   14
    4      1    5    6
    5      3    4   17
    6      4    7   19
    7      6    9   21
    8      1    9   10
    9      7    8   24
   10      8   12   25
   11      2   12   13
   12     10   11   28
   13     11   15   29
   14      3   15   16
   15     13   14   32
   16     14   18   33
   17      5   18   20
   18     16   17   36
   19      6   20   22
   20     17   19   37
   21      7   22   23
   22     19   21   39
   23     21   26   41
   24      9   25   26
   25     10   24   27
   26     23   24   43
   27     25   30   44
   28     12   29   30
   29     13   28   31
   30     27   28   46
   31     29   34   47
   32     15   33   34
   33     16   32   35
   34     31   32   49
   35     33   38   50
   36     18   37   38
   37     20   36   40
   38     35   36   52
   39     22   40   42
   40     37   39   53
   41     23   42   45
   42     39   41   54
   43     26   44   45
   44     27   43   48
   45     41   43   56
   46     30   47   48
   47     31   46   51
   48     44   46   57
   49     34   50   51
   50     35   49   55
   51     47   49   58
   52     38   53   55
   53     40   52   54
   54     42   53   59
   55     50   52   60
   56     45   57   59
   57     48   56   58
   58     51   57   60
   59     54   56   60
   60     55   58   59
