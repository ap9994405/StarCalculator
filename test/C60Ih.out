 12500 + 41580 x + 67650 x^2 + 66900 x^3 + 42000 x^4 + 16352 x^5 + 3620 x^6 + 360 x^7 + 5 x^8
 total: 250967
